plugins.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Plugins
+headline: Pico Plugins
+description: |
+  Want to extend the functionality of Pico?  Below is a collection of our community-developed plugins.<br>
+  Would you like to create your own plugin? Learn how in [our documentation](/docs/#plugins)!
+nav: 5
+
+portfolio:
+  layout: card
+  categories:
+    admin: Administration
+    developer: Developer-Focused
+    theming: Theming Extensions
+  defaultThumbnail: plugins/images/thumbnails/plugin.png
+  defaultImage: themes/images/fullsize/default_theme.png
+
+---
+
+{% include portfolio.html portfolio=site.plugins %}
+
+---
+
+## More Plugins & How to Contribute Your Own
+
+The plugins shown here are only a smaller selection of those available.  You can find more themes on [our Wiki][].
+
+You can even [contribute][] your own plugin to the community and we'll feature it here.  While you can freely add your plugin to [our Wiki][], if you'd also like to be included on this page, you can follow the simple instructions in our [Submission Guidelines][].
```

Comment:
> We should probably change layout to something else.

Yeah, definitely.

> Created a new image for the Contribute tile. Not necessarily 100% final, but any input is welcome.

Looks great! :heart: :smiley:

> on the Plugins page it's a little out of place though (since there's an absolute lack of screenshots/color).

Yeah, maybe a little bit. How about using the default thumbnail's background for plugins?